src/components/ABS_RoomSelectionCarousel/hooks/useCarouselState.ts

```
@@ -92,6 +106,84 @@ const carouselReducer = (state: CarouselState, action: CarouselAction): Carousel
+      const { dragState } = state
+      const deltaX = dragState.deltaX
+      const duration = Date.now() - dragState.startTime
+      const velocity = Math.abs(deltaX) / Math.max(duration, 1)
+      
+      // Determine if we should trigger navigation
+      const threshold = 50 // minimum distance to trigger navigation
+      const velocityThreshold = 0.5 // minimum velocity for quick swipes
+      
+      const shouldNavigate = Math.abs(deltaX) > threshold || velocity > velocityThreshold
+      
+      let newState = {
```

ESLint

[error] 'newState' is never reassigned. Use 'const' instead.

(prefer-const)